checker/manual/external-tools.tex

```diff
@@ -302,17 +302,17 @@ \subsection{Explanation\label{ant-task-explanation}}
 \section{Maven\label{maven}}
```

Comment:
Hi Renato. I tried your instructions on Google Guava ( https://github.com/google/guava ) on guava/pom.xml (I picked guava/pom.xml as it is simpler than the main pom.xml) and indicated that I wanted to build using JDK 7. I noticed that Maven is indeed locating the Nullness Checker because if I misspell the checker's name I get an error. However the Nullness Checker is not being run. I noticed this first because the build time was too short. I then picked a random Java file and introduced an error the nullness checker would catch and running "mvn compile" did not issue any typechecking errors:
import org.checkerframework.checker.nullness.qual.*;
...
@NonNull Object o = null; // Expected to see an error but didn't.

Could you please try it and let me know what I am doing wrong? I can send you my modified pom.xml if you like. Thanks.